app/controllers/whitehall_migration_controller.rb

```diff
@@ -6,8 +6,14 @@ def show
   end
 
   def documents
-    whitehall_migration = WhitehallMigration.find(params[:migration_id])
-    @documents = whitehall_migration.document_imports
+    state_param = params[:state]
+    @whitehall_migration = WhitehallMigration.find(params[:migration_id])
+    if state_param
+      filter = @whitehall_migration.document_imports.where(state: state_param)
+      @documents = filter.order(:id).page(params[:page]).per(25)
+    else
```

Comment:
Do we even show documents without a state anymore? From what I can see the UI only allows users to see a random selection and by 'state' 🤔

Reply:
Hi, they can be viewed without a state ( aka http://content-publisher.dev.gov.uk/whitehall-migration/3/documents) but we don't have a link to view all without state. I don't think we ever explicitly had a link to do so, it was just that we hadn't added the filtering yet